src/shared/ranking.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * This function get data from localStorage by category name.
+ * @param {string} category
+ * Function return [{username: string, points: number}]
+ */
+export function getRankingByCategory(category) {
+  const rankingStorage = window.localStorage.getItem(category);
+  if (rankingStorage == null) {
+    return false;
+  }
+  return rankingStorage;
```

Comment:
Obecnie zwraca stringa (bezpośrednio to co dostaliśmy z localStorage), a nie to co zasugerowałeś że powinno w komentarzu.